supertokens_python/recipe/emailverification/api/email_verify.py

```diff
@@ -40,18 +43,29 @@ async def handle_email_verify_api(
             raise_bad_input_exception("The email verification token must be a string")
 
         token = body["token"]
-        user_context = default_user_context(api_options.request)
+
+        session = await get_session(
+            api_options.request,
+            session_required=False,
+            override_global_claim_validators=lambda _, __, ___: [],
+            user_context=user_context,
+        )
 
         result = await api_implementation.email_verify_post(
-            token, api_options, user_context
+            token, api_options, user_context, session
         )
     else:
         if api_implementation.disable_is_email_verified_get:
             return None
 
-        user_context = default_user_context(api_options.request)
+        session = await get_session(
+            api_options.request,
+            override_global_claim_validators=lambda _, __, ___: [],
+            user_context=user_context,
+        )
+
```

Comment:
how come up do not have to do `assert session is not None`? Cause you do that in generate_email_verify_token.py` file.

Reply:
TODO